src/mbgl/programs/symbol_program.hpp

```diff
@@ -61,8 +61,8 @@ struct SymbolLayoutAttributes : gl::Attributes<
             {{
                 static_cast<int16_t>(labelAnchor.x),
                 static_cast<int16_t>(labelAnchor.y),
-                static_cast<int16_t>(::round(o.x * 64)),  // use 1/64 pixels for placement
-                static_cast<int16_t>(::round((o.y + glyphOffsetY) * 64))
+                static_cast<int16_t>(::round(o.x * 32)),  // use 1/32 pixels for placement
+                static_cast<int16_t>(::round((o.y + glyphOffsetY) * 32))
```

Comment:
this seems to differ from the JS version, but it's not a new difference https://github.com/mapbox/mapbox-gl-js/pull/6375/files#diff-c3312919f069eefcab838ca0389b355aR108 – I think what's happening is you add the offset before `oy` is passed to the js `addVertex` function so they are in fact the same, but just flagging to making sure